00-intro.md

```diff
@@ -11,15 +11,15 @@ minutes: 5
 
 At a high level, computers do four things:
 
--   run programs
--   store data
--   communicate with each other
--   interact with us
+-   run programs,
+-   store data,
+-   communicate with each other,
+-   and interact with us.
```

Comment:
Okay, great, but the goal of style guide is consistency, can you check the other SWC bulleted lists to see how they do things, we're going to stick with that way, whatever it is.